### PR TITLE
Build pipeline fixes

### PR DESCRIPTION
**Pull request by anderseknert**
Get GitHub Actions running before expanding on more tasks, like Docker push.

### CODE REVIEW

**Comment by ashutosh-narkar** [comment]
I think you should be able to test this out in your fork.

File: .github/workflows/pull-request.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      uses: actions/checkout@v2
+
+    - name: Build (Linux)
+      run: make build
+
+    - name: Upload binaries
+      uses: actions/upload-artifact@v2
+      with:
+        name: opa-docker-authz
+        path: ./
```
Comment (srenatus) [inline .github/workflows/pull-request.yaml:21]: If we want to upload just one binary, we should do that... I think `opa-docker-authz` is the collective "artifact name", and with `path: ./` we'll perhaps upload the entire working directory...? I could be wrong 🤔

**Review by srenatus — APPROVED** [review]
Let's merge it and take it from there

File: .github/workflows/pull-request.yaml (hunk 1 above)
Comment (anderseknert) [reply .github/workflows/pull-request.yaml:21]: I think you're right. Might be excessive to upload the binary on each PR too.. I'll remove this for now.

**Pull request merged**